src/main/java/fr/ens/biologie/genomique/eoulsan/modules/diffana/DESeq2Checker.java

```diff
@@ -50,21 +87,23 @@ static boolean checkExperimentDesign(Experiment experiment, boolean throwsExcept
                             + experiment.getName()
                             + " experiment, comparison cannot have more than 1 value: " + c, throwsException);
                 }
-
-                // Get the name of all Metadata keys
-                String[] metakeys = DesignUtils.getExperimentSampleAllMetadataKeys(experiment).toArray(new String[0]);
+                // Get the name of each comparison
+                for (String string : splitC){
+                    if (!string.contains("_vs_")){
+                        comparisionNames.add(string);
+                    }
+                }
```

Comment:
Very bad idea to call a variable "string".
If splitC = "_vs_" exist -> error
If splitC = "toto_vs_" exist -> error
If splitC = "_vs_titi" exist -> error